packages/runtime/src/next/api-handler/index.ts

```diff
@@ -33,6 +33,7 @@ type MakeswiftApiHandlerConfig = {
   appOrigin?: string
   apiOrigin?: string
   getFonts?: GetFonts
+  onPublish?: OnPublish
```

Comment:
I wonder if we should put this under a `webhook` field so that we have future webhook callbacks grouped in there 🤔

Reply:
That's a good idea!